.github/workflows/report.yml

```diff
@@ -168,4 +168,6 @@ jobs:
     uses: ./.github/workflows/deploy-frontend.yml
     with:
       report_artifact_in_scope: true
-    if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
+    if: >
```

Comment:
You can just remove the if entirely now no?

Reply:
okay!